src/features/Courses/CoursesTable/columns.jsx

```diff
@@ -30,6 +37,45 @@ const columns = [
     Header: 'Students invited',
     accessor: 'numberOfPendingStudents',
   },
+  {
+    Header: '',
+    accessor: 'className',
+    cellClassName: 'dropdownColumn',
+    Cell: ({ row }) => {
+      const [isOpenModal, openModal, closeModal] = useToggle(false);
+      const courseDetailsLink = `${getConfig().LEARNING_MICROFRONTEND_URL}/course/${row.original.masterCourseId}/home`;
+      return (
+        <Dropdown className="dropdowntpz">
+          <Dropdown.Toggle
+            id="dropdown-toggle-with-iconbutton"
+            as={IconButton}
+            src={MoreHoriz}
+            iconAs={Icon}
+            variant="primary"
+          />
+          <Dropdown.Menu>
+            <Dropdown.Item href={courseDetailsLink} target="_blank">
```

Comment:
Missing [noopener](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noopener) and [noreferrer](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel/noreferrer)

```suggestion
            <Dropdown.Item href={courseDetailsLink} target="_blank" rel="noopener noreferrer">**
```